compiler/src/iree/compiler/Dialect/HAL/IR/HALBase.td

```diff
@@ -689,6 +689,35 @@ def HAL_DeviceTargetAttr :
     // the `hal.device.targets` attribute is found.
     static SmallVector<DeviceTargetAttr, 4> lookup(Operation *op);
 
+    // Returns true if there is any UnitAttr with |name| in any device
+    // configuration for the given |op|.
+    static bool lookupConfigAttrAny(Operation *op, StringRef name);
```

Comment:
Would adding a String variant later be OK? (e.g., lookupConfigStringAttr or what not), or is it restricted to bool & UnitAttr on purpose?